core/src/main/java/com/linecorp/armeria/client/retry/BackoffWrapper.java

```diff
@@ -0,0 +1,18 @@
+package com.linecorp.armeria.client.retry;
+
+class BackoffWrapper implements Backoff {
```

Comment:
If `BackoffWrapper` is package-local, `protected` could be omitted for its methods. How about making it public?